docs/.vuepress/helpers.js

```diff
@@ -8,7 +8,7 @@ const unsortedVersions = fs.readdirSync(path.join(__dirname, '..'))
 
 const availableVersions = unsortedVersions.sort((a, b) => semver.rcompare(semver.coerce(a), semver.coerce((b))));
 const TMP_DIR_FOR_WATCH = '.watch-tmp';
-const MIN_FRAMEWORKED_DOCS_VERSION = '12.0.1';
+const MIN_FRAMEWORKED_DOCS_VERSION = '11.1.0';
```

Comment:
I'm pretty sure this change was unintentional.

Reply:
> There are still places where the images are still broken. I found these:
> 
> * http://localhost:8080/docs/vue2-data-grid/batch-operations/#overview
> * http://localhost:8080/docs/vue2-data-grid/modules/#about-modules
> * http://localhost:8080/docs/vue2-data-grid/column-moving/#drag-and-move-actions-of-manualcolumnmove-plugin
> * http://localhost:8080/docs/vue2-data-grid/row-moving/#drag-and-move-actions-of-manualrowmove-plugin
> * http://localhost:8080/docs/vue2-data-grid/migration-from-7.4-to-8.0/#changes-in-manualrowmove-and-manualcolumnmove-plugins-behavior

Thanks for that! Fixed within https://github.com/handsontable/handsontable/pull/9482/commits/c7448bf2325017f8ca41048633f2c09ee13e83bf.

Reply:
> I'm pretty sure this change was unintentional.

Sure, changed within: https://github.com/handsontable/handsontable/pull/9482/commits/c7448bf2325017f8ca41048633f2c09ee13e83bf.